include/deal.II/lac/sparse_direct.h

```diff
@@ -512,53 +525,61 @@ class SparseDirectMUMPS : public EnableObserverPointer
       , blr(blr)
     {}
 
-    /*
-     * If true, the MUMPS solver will print out details of the execution.
+    /**
+     * Print details of the execution
      */
     bool output_details;
 
-    /*
-     * If true, the MUMPS solver will print out error statistics.
+    /**
+     * Print error statistics
      */
     bool error_statistics;
```

Comment:
I think the previous text was actually more descriptive (and was grammatically correct, with the period at the end). Did you mean to change this?